ORStools/proc/base_processing_algorithm.py

```diff
@@ -192,11 +192,11 @@ def _get_ors_client_from_provider(provider: str, feedback: QgsProcessingFeedback
     def parseOptions(self, parameters: dict, context: QgsProcessingContext) -> dict:
         options = dict()
 
-        if parameters[self.IN_AVOID_FEATS]:
+        if parameters[self.IN_AVOID_FEATS] is not None:
             options['avoid_features'] = [dict(enumerate(AVOID_FEATURES))[feat] for feat in parameters[self.IN_AVOID_FEATS]]
 
-        if parameters[self.IN_AVOID_BORDERS]:
-            options['avoid_features'] = [dict(enumerate(AVOID_BORDERS))[feat] for feat in parameters[self.IN_AVOID_BORDERS]]
+        if parameters[self.IN_AVOID_BORDERS] is not None:
+            options['avoid_borders'] = dict(enumerate(AVOID_BORDERS))[parameters[self.IN_AVOID_BORDERS]]
```

Comment:
once above 2 are discussed i will integrate this in the actual commit adding this code section

Reply:
see above